src/benchmarks/micro/corefx/System.Runtime/Perf.StringBuilder.cs

```diff
@@ -137,5 +137,38 @@ public void SetupStringBuilderToString()
 
         [Benchmark]
         public string StringBuilderToString() => _bigStringBuilder.ToString();
+
+#if NETCOREAPP3_0 // Append(ReadOnlyMemory<char> value) is available only for .NET Core 3.0+
```

Comment:
Is this actually needed?  Seems like the perf would just be better when built against 3.0, but it should still compile against older versions and just bind to the object-based overload, right?